pkg/scheduler/framework/score_test.go

```diff
@@ -81,15 +146,17 @@ func TestClusterScoreLess(t *testing.T) {
 	}
 }
 
-func TestClusterScoreEqual(t *testing.T) {
+func TestClusterScoreEqualWhenLess(t *testing.T) {
```

Comment:
```suggestion
func TestClusterScoreLess_whenEqual(t *testing.T) {
```
it's testing the less func when the score are equal?

Reply:
Done.